apps/dashboard/widgets.py

```diff
@@ -107,10 +107,16 @@ def render(self, name, value, attrs=None):
         else:
             final_attrs['value'] = ''
 
+        if attrs.get('placeholder', False):
```

Comment:
I think the placeholder bug lies here. You aren't testing if a date is already set, so the placeholder is put in if it exists.